### PR TITLE
Incrementally correcting typos in docs

### PR DESCRIPTION
**Pull request by hunterbarber**
I'll convert out of draft after some time of reviewing documentation to indicate it is ready.

### CODE REVIEW

File: src/opyrability.py [hunk 1]
```diff
@@ -138,7 +138,7 @@ def multimodel_rep(model: Callable[...,Union[float,np.ndarray]],
                                 EDS_resolution=EDS_resolution, 
                                 plot= False)
     else:
-        u0_input = input('Enter an initial estimate for your inverse model'
+        u0_input = input('Enter an initial estimate for your inverse model '
```
Comment (hunterbarber) [inline src/opyrability.py:141]: This is for proper spacing in the prompt

File: src/opyrability.py [hunk 2]
```diff
@@ -43,9 +43,9 @@ def multimodel_rep(model: Callable[...,Union[float,np.ndarray]],
                   resolution: np.ndarray,
                   polytopic_trace: str = 'simplices',
                   perspective: str = 'outputs',
-                  plot: str = True,
-                  EDS_bound: str = None,
-                  EDS_resolution: str = None):
+                  plot: bool = True,
+                  EDS_bound = None,
+                  EDS_resolution = None):
```
Comment (hunterbarber) [inline src/opyrability.py:48]: not sure if you wanted to default as 
```suggestion
                  EDS_bound: np.ndarray = None,
                  EDS_resolution: np.ndarray =  = None):
```

File: src/opyrability.py [hunk 3]
```diff
@@ -43,9 +43,9 @@ def multimodel_rep(model: Callable[...,Union[float,np.ndarray]],
                   resolution: np.ndarray,
                   polytopic_trace: str = 'simplices',
                   perspective: str = 'outputs',
-                  plot: str = True,
-                  EDS_bound: str = None,
-                  EDS_resolution: str = None):
+                  plot: bool = True,
+                  EDS_bound = None,
```
Comment (hunterbarber) [inline src/opyrability.py:47]:
```suggestion
                  EDS_bound: np.ndarray = None,
```

File: src/opyrability.py [hunk 4]
```diff
@@ -1760,7 +1761,7 @@ def implicit_map(model:             Callable[...,Union[float,np.ndarray]],
     jit:  bool True, optional
         JAX's Just-in-time compilation (JIT) of implicit function and its 
         respective multidimensional derivatives (Jacobians). JIT allows faster
-        computation of the implicit map. The default is 'True'.
+        computation of the implicit map. The default is True.
```
Comment (hunterbarber) [inline src/opyrability.py:1764]: The `implicit_map` is missing docstrings for `model`, `image_init`, and `domain_points`

Comment (hunterbarber) [reply src/opyrability.py:1764]: Also having `jit` as the boolean argument and the import (`from jax import jit, jacrev`) is ambiguous whether the code is properly functiong properly

**Review by victoraalves — COMMENTED** [review]
LGTM

**Review by victoraalves — APPROVED** [review]

**Pull request merged**